monitoring/dashboard.py

```diff
@@ -525,6 +557,172 @@ def top10_errors_by_bucket(title, code):
+        average_quota_latency_target(
+            title="Data write (bucket, exceeded)",
+            type='"bucket"', code='"429"'),
+        average_quota_latency_target(
+            title="Data write (account, success)",
+            type='"account"', code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (account, exceeded)",
+            type='"account"', code='"429"'),
+        average_quota_latency_target(
+            title="Data write (account & bucket, success)",
+            type='"bucket+account"',
+            code='~"2.."'),
+        average_quota_latency_target(
+            title="Data write (account & bucket, exceeded)",
+            type='"bucket+account"',
+            code='"429"'),
+        average_quota_latency_target(
+            title="Data deletion", type='"delete"'),
+    ],
+)
+
+
+averageMetricsRetrievalLatencies = TimeSeries(
+    title="Average utilization metrics retrieval latencies",
+    dataSource="${DS_PROMETHEUS}",
+    lineInterpolation="smooth",
+    spanNulls=3*60*1000,
+    unit=UNITS.SECONDS,
+    targets=[
+        average_quota_retrieval_latency(title="Bucket (success)",
+                                        _class='"bucket"', code='~"2.."'),
+        average_quota_retrieval_latency(title="Account (success)",
+                                        _class='"account"', code='~"2.."'),
+        average_quota_retrieval_latency(title="Bucket (error)",
+                                        _class='"bucket"', code='~"4..|5.."'),
+        average_quota_retrieval_latency(title="Account (error)",
+                                        _class='"account"', code='~"4..|5.."'),
+    ],
+)
+
+
+bucketQuotaCounter = Stat(
+    title="Buckets with quota",
+    description=(
+        "Number of S3 buckets with quota enabled in the cluster.\n"
+        "This value is computed asynchronously, and update "
+        "may be delayed up to 1h."
+    ),
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="value",
+    format=UNITS.SHORT,
+    noValue="-",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='max(s3_cloudserver_quota_buckets_count{namespace="${namespace}", job=~"${reportJob}"})',  # noqa: E501
+    )],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("blue", 1, 0.0),
+    ],
+)
+
+
+accountQuotaCounter = Stat(
+    title="Accounts with quota",
+    description=(
+        "Number of accounts with quota enabled in the cluster.\n"
+        "This value is computed asynchronously, and update "
+        "may be delayed up to 1h."
+    ),
+    dataSource="${DS_PROMETHEUS}",
+    colorMode="value",
+    format=UNITS.SHORT,
+    noValue="-",
+    reduceCalc="lastNotNull",
+    targets=[Target(
+        expr='max(s3_cloudserver_quota_accounts_count{namespace="${namespace}", job=~"${reportJob}"})',  # noqa: E501
+    )],
+    thresholds=[
+        Threshold("#808080", 0, 0.0),
+        Threshold("blue", 1, 0.0),
+    ],
+)
+
```

Comment:
`s3_cloudserver_quota_request_with_metrics_unavailable` is maybe the most important metric, should be in dashboard (generally: if you have an alert, you should be able to monitor the metric...)

Reply:
Added as a timeserie (middle-top)
![image](https://github.com/scality/cloudserver/assets/91462779/442961f3-519c-4b73-ad3a-93b5abd861f1)

This directly maps with the alert, as we have one if we have >0 values for 5min (maybe it should be less?)

The description of the critical alert was also updated, to be understandable by someone looking at the platform without context